File: 2018-12-23/2675/2675.java
import java.io.*;

class Main{
	public static void main(String[] args)throws Exception{
		BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in));
		int T = Integer.parseInt(bfr.readLine());
		String[] str;
		int R;
		StringBuilder strBuilder = new StringBuilder();
		for(int i = 0; i < T;i++){
			str = bfr.readLine().split(" ");
			R = Integer.parseInt(str[0]);
			for(int j = 0; j< str[1].length(); j++){
				for(int r = 0; r < R; r++)
					strBuilder.append(str[1].charAt(j));
			}
			strBuilder.append("\n");
		}
		System.out.print(strBuilder);
	}
}

File: 2018-12-23/2675/2675.py
from sys import stdin

T = int(stdin.readline())

for i in range(0, T):
	R, inputData = stdin.readline().split()
	for s in range(0, len(inputData)):
		for r in range(0, int(R)):
			print(inputData[s], end='')
	print()		